js/src/admin/components/FieldAnswersEdit.js

```diff
@@ -14,19 +15,21 @@ export default class FieldAnswersEdit extends Component {
     }
 
     config() {
-        this.$('.js-answers-container')
-            .sortable({
-                handle: '.js-answer-handle',
-            })
-            .on('sortupdate', () => {
-                const sorting = this.$('.js-answer-data')
-                    .map(function () {
-                        return $(this).data('id');
-                    })
-                    .get();
+        let self = this;
```

Comment:
You should use an arrow function below so that `this` can be accessed without needing a temporary variable